groups/primitive-types.yml

```diff
@@ -0,0 +1,2 @@
+name: Primitive types
```

Comment:
We don't have descriptions yet for groups, so I'm going to leave this comment here for clarity and future use.

```suggestion
# JavaScript primitive data types, such as strings or numbers, and their corresponding object wrappers, such as `String` and `Number`.
name: Primitive types
```